plexupdate.sh

```diff
@@ -335,6 +335,11 @@ if [ "${CHECKUPDATE}" = "yes" -a "${AUTOUPDATE}" = "no" ]; then
 fi
 
 if [ "${PUBLIC}" = "no" -a -z "$TOKEN" ]; then
+	declare -f getPlexToken > /dev/null
+	if [ $? -ne 0 ]; then
+		error "getPlexToken() function missing, check that it its sourced correctly"
```

Comment:
All of the error messages in `plexupdate.sh` should be as user-friendly as possible. I'd suggest something that just mentions that the token couldn't be retrieved. And in keeping with the style of the other error messages that we use.